src/evops/metrics/metrics.py

```diff
@@ -66,86 +68,84 @@ def dice(
 
 
 def precision(
-    pred_indices: NDArray[Any, np.int32],
-    gt_indices: NDArray[Any, np.int32],
+    pred_labels: NDArray[Any, np.int32],
+    gt_labels: NDArray[Any, np.int32],
+    statistics: str,
 ) -> np.float64:
     """
     :param pc_points: source point cloud
-    :param pred_indices: indices of points that belong to one plane obtained as a result of segmentation
-    :param gt_indices: indices of points belonging to the reference plane
+    :param pred_labels: labels of points that belong to one planes obtained as a result of segmentation
+    :param gt_labels: labels of points belonging to the reference planes
+    :param statistics: helper function to calculate statistics
     :return: precision metric value for plane
     """
     assert (
-        len(pred_indices.shape) == 1
+        len(pred_labels.shape) == 1
     ), "Incorrect predicted label array size, expected (n)"
     assert (
-        len(gt_indices.shape) == 1
+        len(gt_labels.shape) == 1
     ), "Incorrect ground truth label array size, expected (n)"
-    assert pred_indices.size != 0, "Predicted indices array size must not be zero"
-
-    return __precision(pred_indices, gt_indices)
-
-
-def accuracy(
-    pred_indices: NDArray[Any, np.int32],
-    gt_indices: NDArray[Any, np.int32],
-) -> np.float64:
-    """
-    :param pc_points: source point cloud
-    :param pred_indices: indices of points that belong to one plane obtained as a result of segmentation
-    :param gt_indices: indices of points belonging to the reference plane
-    :return: accuracy metric value for plane
-    """
-    assert (
-        len(pred_indices.shape) == 1
-    ), "Incorrect predicted label array size, expected (n)"
+    assert pred_labels.size != 0, "Predicted labels array size must not be zero"
+    statistics_functions = {"iou": __iou_overlap}
```

Comment:
why do we define this dict here? I think it have to be placed in the root of file or even in the extra file